src/overlays/actors/ovl_En_Test3/z_en_test3.c

```diff
@@ -27,113 +203,1114 @@ const ActorInit En_Test3_InitVars = {
+                                                    ((play->actorCtx.unk5 & 0x20) || (play->actorCtx.unk5 & 0x10)))) {
+        if (this->player.csMode != 5) {
+            play->startPlayerCutscene(play, &this->player, 5);
+        }
+        play->actorCtx.unk5 &= ~0x10;
+    } else if (this->player.actor.category == ACTORCAT_PLAYER) {
+        func_80A409D4(this, play);
+    } else if (play->startPlayerCutscene(play, &this->player, 0)) {
+        if (this->unk_D88 >= 7) {
+            Vec3f worldPos;
+
+            Math_Vec3f_Copy(&worldPos, &this->player.actor.world.pos);
+            this->unk_D80 = 4;
+            func_80A40230(this, play);
+            Math_Vec3f_Copy(&this->player.actor.world.pos, &worldPos);
+            D_80A41D48 = false;
+            this->unk_D84 = 0.0f;
+        }
+    } else {
+        D_80A41D40 = 0.0f;
+        D_80A41D44 = this->player.actor.shape.rot.y;
+        this->unk_D94(this, play);
+        sInput.press.button = (sInput.rel.button ^ sInput.cur.button) & sInput.cur.button;
+        func_800B6F20(play, &sInput, D_80A41D40, D_80A41D44);
+    }
+    play->playerUpdate(&this->player, play, &sInput);
+    if (D_80A41D48) {
+        this->player.actor.world.pos.x = D_80A41D50.x;
+        this->player.actor.world.pos.z = D_80A41D50.z;
+        this->player.linearVelocity = 0.0f;
+    }
+}
+
+static s32 D_80A418C8 = false;
+
+s32 EnTest3_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnTest3* this = THIS;
+
+    if (limbIndex == OBJECT_TEST3_LIMB_01) {
+        D_80A41D6C = &this->player.meleeWeaponInfo[2].base;
+        if (!(this->player.skelAnime.moveFlags & ANIM_FLAG_4) || (this->player.skelAnime.moveFlags & ANIM_FLAG_1)) {
+            pos->x *= this->player.ageProperties->unk_08;
+            pos->z *= this->player.ageProperties->unk_08;
+        }
+        if (!(this->player.skelAnime.moveFlags & ANIM_FLAG_4) ||
+            (this->player.skelAnime.moveFlags & ANIM_FLAG_UPDATEY)) {
+            pos->y *= this->player.ageProperties->unk_08;
+        }
+        pos->y -= this->player.unk_AB8;
+        if (this->player.unk_AAA != 0) {
+
+            Matrix_Translate(pos->x, ((Math_CosS(this->player.unk_AAA) - 1.0f) * 200.0f) + pos->y, pos->z,
+                             MTXMODE_APPLY);
+            Matrix_RotateXS(this->player.unk_AAA, MTXMODE_APPLY);
+            Matrix_RotateZYX(rot->x, rot->y, rot->z, MTXMODE_APPLY);
+            func_80125318(pos, rot);
+        }
+    } else {
+        if (*dList != NULL) {
+            D_80A41D6C++;
+        }
+        if (D_80A418C8) {
+            *dList = NULL;
+        }
+        if (limbIndex == OBJECT_TEST3_LIMB_0B) {
+            rot->x += this->player.unk_AAC.z;
+            rot->y -= this->player.unk_AAC.y;
+            rot->z += this->player.unk_AAC.x;
+        } else if (limbIndex == OBJECT_TEST3_LIMB_0A) {
+            s32 pad; // Must exist here to match
```

Comment:
```suggestion
            s32 pad;
```
This is pretty common, I don't think we need to comment on it every time

Reply:
What's weird is that it must be in that exact scope. Any other scope won't match.

Reply:
Just look up `requiredScopeTemp`, they're everywhere in the repo, where a pad needs to be in an exact scope location. In fact, most of OoT retail is going to need this exact solution, moving temps/pads around so they're in the right scope.

Reply:
I suppose I could just rename it to that

Reply:
I think that would be best, yeah. That tends to be exactly what we call these specific types of pads.